audit/projects/k8s-staging-capi-docker/iam.json

```diff
@@ -20,6 +20,24 @@
       ],
       "role": "roles/cloudbuild.serviceAgent"
     },
+    {
```

Comment:
See comment on compute.  I suspect the same, but I have less evidence).  We should nuke it.

Reply:
added to https://github.com/kubernetes/k8s.io/issues/1675